submissions.md

```diff
@@ -3,18 +3,63 @@ layout: default
 ---
 
-# Proceedings of ProbNum25
+
+# Call for Papers
+
+_Full details, including style files, will be published here in due course._
+
+## Proceedings of ProbNum25
 
 ProbNum25 will have a proceedings issue, published in the [Proceedings of Machine Learning Research (PMLR)](https://proceedings.mlr.press).
 Talks at ProbNum25 will be selected from among the submissions.
 Submissions can be up to 8 pages, with subjects broadly in the area of probabilistic computation.
 We also welcome and encourage shorter papers.
 
+## Important dates
 Timescales for submitting are anticipated to be as follows, though these are subject to revision:
 
 - **Submission Deadline** 5th March 2025.
 - **Reviews Released** 9th April 2025.
 - **Responses Due** 18 April 2025.
 - **Decisions** 14 May 2025.
 
-Full details, including style files, will be published here in due course.
+
+## Subject areas
+
+We welcome submissions in areas including but not limited to:
+
+- Probabilistic numerical methods
+- Applications of probabilistic numerics
+- Decision-making under uncertainty
+- Uncertainty quantification
+- Spatiotemporal modeling
+- Sequential experimental design
+
+## Submission instructions
+
+**Page limit**  
+Submissions are full papers limited to up **8 pages** excluding references, acknowledgements, and appendices.
+Shorter submissions are very welcome and will be equally considered.
+
+**Formatting instructions**  
+Submissions must use the **ProbNum LaTeX style file** which will be provided shortly here.
+Please do not modify the style file.
+Formatting instructions are available in the sample paper.
+
+**Anonymization**  
+The ProbNum25 review process is double-blind. All submissions must be anonymized and may not contain any information that can violate the double-blind reviewing policy, such as the author names or their affiliations, acknowledgements, or links that can infer any author’s identity or institution. Self-citations are allowed as long as anonymity is preserved.
+
+**Submission page**  
+The submission will be done via OpenReview. The submission page will be available by early 2025.  
+Please upload a single file; you can either submit a single pdf file or a single zip file for further supplementary material in other formats.
+
+**Dual submissions**  
+Submitted manuscripts should not have been previously published in a journal or in the proceedings of a conference, and should not be under consideration for publication at another conference at any point during the ProbNum25 review process. This excludes non-archival venues such as workshops.
+
+**Confidentiality**  
+Reviewer will be instructed to keep them confidential during the review process and delete them once the review process has concluded.
+
+
+## Reviewer nomination
+For each submission, the authors will be expected to nominate at least one of the authors as a reviewer for ProbNum25. Nominated reviewers are expected to have sufficient expertise in the relevant field.
+
```

Comment:
nominated authors must make a good faith effort to provide timely, and high-quality reviews. Unprofessional conduct of a reviewer can be grounds for desk-rejection of papers submitted by the person in question.